Setup/update.sh

```diff
@@ -162,17 +177,18 @@ function get_latest_version() {
     fi
 
     echo "$VERSION_FILE_PATH"
-    curl -s -S -o "$ES_VER_FILE" "$VERSION_FILE_PATH"
+    curl -s -S -o "$ES_VER_FILE_NEW" "$VERSION_FILE_PATH"
```

Comment:
I would like to pay attention that we download shield-versions.txt at least 3 times during execute this script. It's not reason for damage and inordinate traffic, but any way...